body_to_camera: {
  qx: 0.4951858815239740,
  qy: 0.4783353420041345,
  qz: 0.5127316954379839,
  qw: 0.5129253862136314,
  tx: 0.0674729117597292,
  ty: -0.0016263140276227,
  tz: 0.0894389736886632 }

